### ISSUE TITLE
Manage Once in OS Sierra

### ISSUE THREAD

**Issue by gmillercmsd** [timsutton/mcxToProfile#16]
Manage Once does not seem to apply to users on Sierra when deployed through Casper.

**Comment by macmule** [comment]
@gmillercmsd This is more than likely a jamf issue.

For years, the jss's login hook has refreshed MCX at login, as shown below:

![screenshot 2016-09-21 16 16 58](https://cloud.githubusercontent.com/assets/2464974/21958004/8c555984-da9b-11e6-9018-5b06770a93c2.png)

Which lead to the following FR from me: [Allow policy execution once per computer, per user](https://www.jamf.com/jamf-nation/feature-requests/45/allow-policy-execution-once-per-computer-per-user)

It's also possibly that the preferences system in Sierra has changed so that some keys are only respected when Forced & not set once. I mentioned this during my JNUC2016 talk on profiles: [Profiles: An IT Admin’s Best Friend | JNUC 2016](https://www.youtube.com/watch?v=JbukZ0SNafY)

In short, I don't think MCXToProfile can help here due to either of the above. Maybe [Outsets login-once](https://github.com/chilcote/outset/wiki/FAQ#what-happens-during-login) could be used instead?

**Comment by alifkifi** [comment]
ali
